### PR TITLE
[plugin.video.gamekings] 1.2.1

### PR DESCRIPTION
**Pull request by skipmodea1**

### CODE REVIEW

File: plugin.video.gamekings/resources/language/Dutch/strings.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <string id="30001">TV Afleveringen</string>
```
Comment (phil65) [inline plugin.video.gamekings/resources/language/Dutch/strings.xml]: please remove the language xmls.

File: plugin.video.gamekings/resources/lib/gamekings_list.py [hunk 2]
```diff
@@ -234,85 +198,53 @@ def getVideos(self):
             if title[0:1] == " ":
                 title = title[1:]
 
+            title = str(title).replace("aflevering", "Aflevering")
```
Comment (phil65) [inline plugin.video.gamekings/resources/lib/gamekings_list.py]: "Aflevering" should probably get translated?

File: plugin.video.gamekings/resources/lib/gamekings_search.py [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+            title = title.replace(' xxvii ', ' XXVII ')
+            title = title.replace(' xxviii ', ' XXVIII ')
+            title = title.replace(' xxix ', ' XXIX ')
+            title = title.replace(' xxx ', ' XXX ')
+
+            # remove space on first position in the title
+            if title[0:1] == " ":
+                title = title[1:]
+
+            title = str(title).replace("aflevering", "Aflevering")
```
Comment (phil65) [inline plugin.video.gamekings/resources/lib/gamekings_search.py]: same here

File: plugin.video.gamekings/resources/language/Dutch/strings.xml (hunk 1 above)
Comment (skipmodea1) [inline plugin.video.gamekings/resources/language/Dutch/strings.xml]: Done!

File: plugin.video.gamekings/resources/lib/gamekings_list.py (hunk 2 above)
Comment (skipmodea1) [inline plugin.video.gamekings/resources/lib/gamekings_list.py]: Done!

File: plugin.video.gamekings/resources/lib/gamekings_search.py (hunk 3 above)
Comment (skipmodea1) [inline plugin.video.gamekings/resources/lib/gamekings_search.py]: Done! and commited with some squashing.

**Pull request merged**